sdk/formrecognizer/ai-form-recognizer/README.md

```diff
@@ -0,0 +1,477 @@
+- `beginComposeModel` starts an operation to compose multiple models into a single model. When used for custom form recognition, the new composed model will first perform a classification of the input documents to determine which of its submodels is most appropriate.
+- `beginCopyModel` starts an operation to copy a custom model from one Form Recognizer resource to another (or even to the same Form Recognizer resource). It requires a `CopyAuthorization` from the target Form Recognizer resource, which can be generated using the `getCopyAuthorization` method.
+- `getInfo` retrieves information about the resource's limits, such as the number of custom models and the maximum number of models the resource can support.
+- `getModel`, `listModels`, and `deleteModel` enable managing models in the resource.
+- `getOperation` and `listOperations` enable viewing the status of model creation operations, even those operations that are ongoing or that have failed. Operations are retained for 24 hours.
+
+Please note that models can also be created using the Form Recognizer service's graphical user interface: [Form Recognizer Labeling Tool][fr-labeling-tool].
+
+Sample code snippets that illustrate the use of `DocumentModelAdministrationClient` to build a model can be found [below, in the "Build a Model" example section.](#build-a-model).
+
+### Long-Running Operations
+
+Long-running operations (LROs) are operations which consist of an initial request sent to the service to start an operation, followed by polling for a result at a certain interval to determine if the operation has completed and whether it failed or succeeded. Ultimately, the LRO will either fail with an error or produce a result.
+
+In Azure Form Recognizer, operations that create models (including copying and composing models) as well as the analysis/data-extraction operations are LROs. The SDK clients provide asynchronous `begin<operation-name>` methods that return `Promise<PollerLike>` objects. The `PollerLike` object represents the operation, which runs asynchronously on the service's infrastructure, and a program can wait for the operation to complete by calling and awaiting the `pollUntilDone` method on the poller returned from the `begin<operation-name>` method. Sample code snippets are provided to illustrate using long-running operations in the next section.
+
+## Examples
+
+The following section provides several JavaScript code snippets illustrating common patterns used in the Form Recognizer client libraries.
+
+- [Analyze a Document with a Custom Model](#analyze-a-document-with-a-custom-model)
+- [Extract Layout](#extract-layout)
+- [Use Prebuilt Models](#using-prebuilt-models)
+- [Build a Model](#build-a-model)
+- [List All Models](#list-all-models)
```

Comment:
Paul suggested to re-ordering the examples in the python README

Reply:
Hm, I really think `beginAnalyzeDocuments` ought to come first, based on my understanding of the customer use cases. I'll reorder a few of mine and extend a couple of these to make them more comprehensive, but for the sake of the flow of the document in JS I'm going to leave `beginAnalyzeDocuments` at the top for now and revisit this in the future.